src/components/explorer.ts

```diff
@@ -4,11 +4,13 @@ import TypedReact = require("typed-react");
 
 import AuthorizedClient = require("../authorized_client");
 import JsonResponse = require("./json_response");
+import RouteEntry = require("./route_entry");
 
 var r = react.DOM;
 
 export interface Props {
-    initial_authorized_client?: AuthorizedClient
+    initial_authorized_client?: AuthorizedClient;
+    initial_route?: string;
```

Comment:
Good call! I fixed `initial_authorized_client` and `authorized_client`. I think `initial_route` is okay, though, since it's just a string? I also switched to using 2-spaces; not sure why I didn't do this from the start.